lib/extract/standalone/nemo/Makefile

```diff
@@ -72,11 +73,17 @@ psy_data_base.f90: $(PSYDATA_LIB_DIR)/psy_data_base.jinja Makefile
 extract_standalone_base.f90: $(LIB_TMPLT_DIR)/extract_standalone_base.jinja Makefile
 	$(PROCESS) $(PROCESS_ARGS) -generic-provide $< > extract_standalone_base.f90
 
+compare_variables_mod.F90: $(PSYDATA_LIB_DIR)/extract/compare_variables_mod.jinja Makefile
```

Comment:
I had a quick look, while there is indeed duplicated code, I think some of the Makefile would be harder to understand if common rules are extracted into the baseclass. E.g. it would not be obvious where PSyclone is triggered etc. But an happy to have a go at a later stage.